lib/board.rb

```diff
@@ -39,11 +39,14 @@ def empty?(spot)
   end
 
   def create_board(board_size)
+    id = 0
+
     new_board = Array.new
     board_size.times do
       new_row = Array.new
       board_size.times do
-        new_row.push(Mark::EMPTY)
+        new_row.push(id)
```

Comment:
It's the value displayed to the user (make me realize it shouldn't be the responsibility of the board).

I don't know much Ruby but I would avoid to use nil as a value. I should looking to that